systers_portal/meetup/views.py

```diff
@@ -31,6 +31,12 @@ class MeetupLocationList(ListView):
     model = MeetupLocation
     paginate_by = 20
 
+    def get_context_data(self, **kwargs):
+        context = super(MeetupLocationList, self).get_context_data(**kwargs)
+        context['meetup_list'] = Meetup.objects.filter(
+            date__gte=datetime.date.today()).order_by('date', 'time')
```

Comment:
Django `datetime.date.today()` cause issues w/ timezones, did you test w/ setting a different timezone in your machine?